open-reac/src/main/java/com/powsybl/openreac/parameters/input/json/VoltageLimitOverrideDeserializer.java

```diff
@@ -26,23 +26,33 @@ public VoltageLimitOverrideDeserializer() {
 
     @Override
     public VoltageLimitOverride deserialize(JsonParser parser, DeserializationContext deserializationContext) throws IOException {
-        double deltaLowVoltageLimit = 0;
-        double deltaHighVoltageLimit = 0;
+        String voltageLevelId = null;
+        VoltageLimitOverride.VoltageLimitType type = null;
+        boolean isRelative = true;
```

Comment:
You might also want to initialize this boolean to null, for that you can use Boolean instead. Or do you want isRelative to have a default value to true if the field is not in the file? 
Then you can add a check that isRelative is well defined and if not throw an exception because it's undefined.

Reply:
ok great, you are right.